hacking/aws_config/testing-iam-policy.json.j2

```diff
@@ -0,0 +1,246 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "AllowDescribeAllEC2",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:Describe*"
+            ],
+            "Resource": [
+                "arn:aws:ec2:{{aws_region}}:{{aws_account}}:*"
+            ]
+        },
+        {
+            "Sid": "MiscPrivilegesNeededByEC2Tests",
```

Comment:
Grumble grumble - I really think this should be scoped to something other than "misc"